ansible/roles/epics-modules/tasks/build_module.yml

```diff
@@ -54,26 +53,29 @@
     remote_src: true
     extra_opts:
       - --transform
-      - "s|^[^/]*|{{ item }}-{{ epics_modules[item].version }}|"    # normalize source path
+      - "s|^[^/]*|{{ item }}-{{ epics_modules[item].version }}|" # normalize source path
   become: true
   when: epics_module_flag.stat.exists == false
 
 - name: "{{ epics_modules[item].name }} | Remove SUPPORT definition from configure/RELEASE"
   ansible.builtin.lineinfile:
     path: "{{ epics_install_path }}/{{ item }}-{{ epics_modules[item].version }}/configure/RELEASE"
     state: absent
-    regexp: '^SUPPORT='
+    regexp: "^SUPPORT="
   become: true
 
 - name: "{{ epics_modules[item].name }} | Run pre-build hook"
   include_tasks: "{{ epics_modules[item].pre_hook }}"
   when: epics_modules[item].pre_hook is defined
 
 - name: "{{ epics_modules[item].name }} | Build from sources (this may take a while)"
-  command: make -C "{{ epics_install_path }}/{{ item }}-{{ epics_modules[item].version }}" all clean
+  command: make {{ epics_modules[item]. args | default('') }} -C "{{ epics_install_path }}/{{ item }}-{{ epics_modules[item].version }}" all clean
```

Comment:
`epics_modules[item]. args` That space looks suspicious.

Reply:
for sure. I wonder why it works?
![image](https://github.com/user-attachments/assets/fa4ae923-210c-479e-bc38-fb7f011ca3af)

Reply:
Probably the parser is very forgiving.